tor_core/helpers.py

```diff
@@ -6,7 +6,7 @@
 
 import praw
 import prawcore
-from slackclient import SlackClient
+import requests
```

Comment:
I believe `requests` still has to be added to the install requirements.

Reply:
Requests is installed as part of PRAW, but yes, it needs to be added to the requirements. Good catch!

Reply:
This has been fixed!